.github/workflows/frontend_checks.yml

```diff
@@ -61,23 +60,32 @@ jobs:
 
       - name: Install dependencies
         run: |
-          cd clients/admin-ui
+          cd clients
           npm install
 
-      - name: Cypress E2E tests
+      - name: Cypress E2E Admin-UI
         uses: cypress-io/github-action@v4
         with:
-          working-directory: clients/admin-ui
+          working-directory: clients
           install: false
-          start: npm run cy:start
+          start: npm run cy-ui:start
           wait-on: "http://localhost:3000"
           wait-on-timeout: 180
 
+      - name: Cypress E2E Privacy-Center
+          uses: cypress-io/github-action@v4
+          with:
+            working-directory: clients
+            install: false
+            start: npm run cy-pc:start
+            wait-on: "http://localhost:3000"
+            wait-on-timeout: 180
+
       - uses: actions/upload-artifact@v3
         if: failure()
         with:
           name: cypress-videos
-          path: /home/runner/work/fides/fides/clients/admin-ui/cypress/videos/*.mp4
+          path: /home/runner/work/fides/fides/clients/cypress-e2e/videos/*.mp4
```

Comment:
oh.. I see, let me look into how we're adding the vids for the other clients, and get these actions added.

Reply:
I don't think we upload from the other clients. I think this was put in originally because we were debugging something specific to `admin-ui`. it's rare to need to debug something that is only failing in CI, so we never bothered to put it in for the other clients.

ideally, instead of using this `upload-artifact` workaround, we should configure cypress dashboard for all of the clients, since dashboard automatically records videos. but I think we'd need more than the free plan to do that.

for now, maybe we can leave it with the old path and expect only admin-ui uploads? or was there a reason for the change to `cypress-e2e`?